addons/binding/org.openhab.binding.airquality/src/main/java/org/openhab/binding/airquality/internal/json/AirQualityJsonTime.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.airquality.internal.json;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * The {@link AirQualityJsonTime} is responsible for storing
+ * the "time" node from the waqi.org JSON response
+ *
+ * @author Kuba Wolanin - Initial contribution
+ */
+public class AirQualityJsonTime {
+
+    private String s;
+    private String tz;
+
+    private final static SimpleDateFormat SDF = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+
+    /**
+     * Get Timezone in following format: "+0100"
+     *
+     * @return {String}
+     */
+    public String getTz() {
+        return tz.replace(":", "");
```

Comment:
Well, It's a good practice for classes [to be responsible for one thing](https://en.wikipedia.org/wiki/Single_responsibility_principle). 
In this case @splatch moved all JSON parsing-related logic to `org.openhab.airquality.intrernal.json` package. 
In `Handler` itself, we're executing Gson so it knows where to put all data
This gives ability to scale the classes in case of e.g. API change or new entry points coming with the payload :)
Cheers